Update dlp-conditions-and-exceptions.md (#7163)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/dlp-conditions-and-exceptions.md

### Senders

> For conditions and exceptions that examine the sender's address, you can specify where rule looks for the sender's address. The sender address used by Data Loss Prevention (DLP) to evaluate sender based conditions listed below is the Envelope address. Note that Exchange transport rules (ETR) uses the header address as the default sender address location.

Reviewer: ```suggestion
For conditions and exceptions that examine the sender's address, you can specify where the rule looks for the sender's address. The sender address, used by the DLP rule to evaluate the sender-based conditions listed below, is the Envelope address. Note that Exchange transport rules use the Header address as the default sender address location.
```

<br>

> >[!Note]
> >Starting 20th January 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.

Reviewer: ```suggestion
> [!NOTE]
> Starting January 20, 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.
```

![image](https://user-images.githubusercontent.com/53205984/145942298-6b435ba6-d146-44fe-a1c5-58babeaf8d7a.png)

> At the tenant level, you can configure a Sender Address Location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate sender address from Envelope across all rules, you can run the below command-

Reviewer: ```suggestion
At the tenant level, you can configure a sender address location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate the sender address from the Envelope across all rules, you can run the following command:
```

```PowerShell
Set-PolicyConfig –SenderAddressLocation Envelope
```

> To configure sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:

Reviewer: ```suggestion
To configure the sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
```

- **Header**: Only examine senders in the message headers (for example, the **From**, **Sender**, or **Reply-To** fields). This is the default value.

- **Envelope**: Only examine senders from the message envelope (the **MAIL FROM** value that was used in the SMTP transmission, which is typically stored in the **Return-Path** field).

- **Header or envelope** (`HeaderOrEnvelope`) Examine senders in the message header and the message envelope.
<br>

> ****

Reviewer: ```suggestion
```
